docs/tutorial/tutorial-5/android.rst

```diff
@@ -2,9 +2,382 @@
 Tutorial 5 - Taking it mobile: Android
 ======================================
 
-To deploy our applicaton to Android...
+The process of deploying an application to Android is very similar to the
+process for deploying as a desktop application. There is one additional
+pre-requisite: Java 8.
+
+Setting up Java 8
+=================
+
+To use BeeWare's Android tooling, you must install a Java 8 compiler and provide
+the path to it in the ``JAVA_HOME`` environment variable. This is because
+BeeWare's Android tooling depends on the Android SDK, and the Android SDK
+depends on the Java runtime and compiler being available. Some tools within
+the Android SDK specifically require version 8 of Java, so we stick to that
+in this documentation.
+
+Download & install Java 8
+-------------------------
+
+Visit the `AdoptOpenJDK website <https://adoptopenjdk.net/>`_ and
+download OpenJDK 8.
+
+.. tabs::
+
+  .. group-tab:: macOS
+
+    On macOS, the default format is a *.pkg* installer. Download and execute the
+    installer, accepting the defaults. In your terminal, run:
+
+    ``export JAVA_HOME="$(/usr/libexec/java_home -v 1.8)"``
+
+    You will need to run this every time you start a terminal session, or you
+    can add it to your `shell configuration <https://apple.stackexchange.com/a/356455>`_
+    or `launchd configuration <http://www.dowdandassociates.com/blog/content/howto-set-an-environment-variable-in-mac-os-x-launchd-plist/>`_.
+
+  .. group-tab:: Linux
+
+    On Linux, the default format is a *.tar.gz* file. Download it and unpack it to a
+    directory of your choosing. Set the *JAVA_HOME* environment variable to the
+    directory containing *bin*; for example, if you unpacked the *.tar.gz* file
+    into ``~/java/1.8/``, at the time of writing, you would need to run:
+
+    ``export JAVA_HOME=$HOME/java/1.8/jdk8u242-b08``
+
+    You will need to run this every time you start a terminal session, or you
+    can add it to your `PAM environment file <https://help.ubuntu.com/community/EnvironmentVariables#Persistent_environment_variables>`_,
+    `shell configuration <https://apple.stackexchange.com/a/356455>`_,
+    or `systemd configuration <https://in.waw.pl/~zbyszek/blog/environmentd.html>`_.
```

Comment:
Same rules here about preferred sources.

Reply:
Fair enough! Maybe the next step can be for us to collaborate over a low-latency mode like chat or a call to avoid long round-trips getting the best references.